docassemble/AssemblyLine/data/questions/al_visual.yml

```
@@ -193,13 +193,12 @@ question: |
 fields:
   - What do you want to do?: al_sharing_type
     datatype: radio
-    choices:
-      - Tell a friend about this website: tell_friend
-      - Share my answers and progress with someone: share_answers
+    code: |
+      sharing_choices
   -  How do you want to share the link?: al_how_share_link
      datatype: radio
      choices:
-       - Email or text message: email_or_sms
+       - Email${ " or text message" if is_sms_enabled() else ""}: email_or_sms
```

```suggestion
       - label: |
           % if is_sms_enabled():
           Email or text message
           % else:
           Email
           % endif
         field: email_or_sms
```

Let's avoid ternary statements for user-facing text--languages change word order, and this kind of expression is going to be mis-read by the translator. See: https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/coding_style_guide/yaml_translation#avoid-using-the-language-functions-from-patternen-in-your-interview